website/profile/views.py

```diff
@@ -514,11 +514,11 @@ def serialize_schools(auth, uid=None, **kwargs):
 def unserialize_names(**kwargs):
     user = kwargs['auth'].user
     json_data = escape_html(request.get_json())
-    user.fullname = json_data.get('full')
-    user.given_name = json_data.get('given')
-    user.middle_names = json_data.get('middle')
-    user.family_name = json_data.get('family')
-    user.suffix = json_data.get('suffix')
+    user.fullname = (json_data.get('full') or '').strip()
```

Comment:
I was originally doing that but changed it when I saw related functionality tests like social specifically testing for None, could switch back if we are ok with guaranteeing a empty string in all cases. (didn't want to just assume)